icons/table.svg

```diff
@@ -9,8 +9,5 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <path d="M12 3v18" />
-  <rect width="18" height="18" x="3" y="3" rx="2" />
-  <path d="M3 9h18" />
-  <path d="M3 15h18" />
+  <path d="M9 3H5a2 2 0 0 0-2 2v4m6-6h10a2 2 0 0 1 2 2v4M9 3v18m0 0h10a2 2 0 0 0 2-2V9M9 21H5a2 2 0 0 1-2-2V9m0 0h18" />
```

Comment:
Not sure if we should see this as the most generic icon for tables.

If I search on google, it looks more like the `table-3x2` is more the generic one.
<img width="715" alt="image" src="https://github.com/lucide-icons/lucide/assets/11825403/620af112-ec4a-462e-b9a8-45a261c23414">